progfun/tutoriales/Haskell/Haskell.html

```diff
@@ -1,361 +1,558 @@
+                  Tiene pocas palabras clave. Usa indentación como medio de
+                  estructuración del código
+                </p>
+              </div>
             </div>
-          </div>
 
-          
-          <div class="col-lg-4 col-md-6 wow fadeInUp" data-wow-delay="0.4s">
-            <div class="box">
-              <h4 class="title"><a href="https://www.haskell.org/">Sistema de tipos</a></h4>
-              <p class="description">
-                Tiene un sistema de tipo que requiere un poco de información del programador con los cuales puede detectar una gran variedad de errores de incompatibilidad.
-              </p>
+            <div class="col-lg-4 col-md-6 wow fadeInUp" data-wow-delay="0.4s">
+              <div class="box">
+                <h4 class="title">
+                  <a href="https://www.haskell.org/">Sistema de tipos</a>
+                </h4>
+                <p class="description">
+                  Tiene un sistema de tipo que requiere un poco de información
+                  del programador con los cuales puede detectar una gran
+                  variedad de errores de incompatibilidad.
+                </p>
+              </div>
             </div>
-          </div>
-          <div class="col-lg-4 col-md-6 wow fadeInUp" data-wow-delay="0.6s">
-            <div class="box">
-              <h4 class="title"><a href="https://www.haskell.org/">Efectos monádicos</a></h4>
-              <p class="description">
-                Para una función dada una misma entrada, siempre se producirá la misma salida, independiemtemente del contexto de las variables.
-              </p>
+            <div class="col-lg-4 col-md-6 wow fadeInUp" data-wow-delay="0.6s">
+              <div class="box">
+                <h4 class="title">
+                  <a href="https://www.haskell.org/">Efectos monádicos</a>
+                </h4>
+                <p class="description">
+                  Para una función dada una misma entrada, siempre se producirá
+                  la misma salida, independiemtemente del contexto de las
+                  variables.
+                </p>
+              </div>
             </div>
-          </div>
-          <div class="col-lg-4 col-md-6 wow fadeInUp" data-wow-delay="0.6s">
-            <div class="box">
-              <h4 class="title"><a href="https://www.haskell.org/">Concurrente</a></h4>
-              <p class="description">
-                Permite que, durante un periodo de tiempo, más de un proceso se ejecute. Esto dado a su compilador que permite la copmutación en paralelo.
-                .
-              </p>
+            <div class="col-lg-4 col-md-6 wow fadeInUp" data-wow-delay="0.6s">
+              <div class="box">
+                <h4 class="title">
+                  <a href="https://www.haskell.org/">Concurrente</a>
+                </h4>
+                <p class="description">
+                  Permite que, durante un periodo de tiempo, más de un proceso

```

Comment:
```suggestion
                  Permite que, durante un cierto tiempo, más de un proceso

```
Eliminar redundancia.